### PR TITLE
Add return description

### PR DESCRIPTION
**Pull request by rbruggem**
Describe the object type that is returned.

### CODE REVIEW

File: src/components/Endpoint/index.tsx [hunk 1]
```diff
@@ -15,6 +15,11 @@ import {
 } from 'lune-ui-lib'
 import React from 'react'
 
+function formatReturnResource(resource: string): string {
+    const article = ['a', 'e', 'i', 'o', 'u'].includes(resource[0].toLowerCase()) ? 'an': 'a'
```
Comment (jstasiak) [inline src/components/Endpoint/index.tsx:19]: Haha, I see what you're doing here. Wondering what can we do to make it right in all cases, like user (now: "an user" but we want "a user")

**Review by jstasiak — APPROVED** [review]

File: src/components/Endpoint/index.tsx (hunk 1 above)
Comment (rbruggem) [reply src/components/Endpoint/index.tsx:19]: You are right.

"An is used before a noun that starts with a vowel sound (e.g., “a,” “o,” “i”)."

Comment (jstasiak) [reply src/components/Endpoint/index.tsx]: Yeah it's a bit more complicated. "A user" but "an underground" (it's the sound that matters)

Comment (jstasiak) [reply src/components/Endpoint/index.tsx]: Or: "an hour" vs "a hammer"

**Comment by rbruggem** [comment]
oops

**Comment by rbruggem** [comment]
Mistake

**Comment by rbruggem** [comment]
Re-opening.

**Pull request closed without merging**